include/pmacc/Environment.hpp

```diff
@@ -456,14 +456,14 @@ namespace detail
 
 
         int maxTries = num_gpus;
-
+#if (PMACC_CUDA_ENABLED == 1)
         cudaDeviceProp devProp;
-        cudaError rc;
-        CUDA_CHECK(cudaGetDeviceProperties(&devProp, deviceNumber));
-
+        CUDA_CHECK((cuplaError_t)cudaGetDeviceProperties(&devProp, deviceNumber));
         /* if the gpu compute mode is set to default we use the given `deviceNumber` */
         if (devProp.computeMode == cudaComputeModeDefault)
             maxTries = 1;
+#endif
+        cudaError rc;
```

Comment:
Must be `cudaError` up to the full port to cupla. Currently we are using all cuda namings and will switch to cupla later on.